polyglot/piranha/src/cleanup_rules/go/scope_config.toml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[scopes]]
+name = "File"
+[[scopes.rules]]
+matcher =  """
+(source_file) @source_file
+"""
+generator = """(source_file) @sf"""
+
+[[scopes]]
+name = "Function-Method"
```

Comment:
are there constructors in Go ? If so, please look at how I have handled constructors in Java/Kotlin

Reply:
I'm not really a Gopher :D, but as far as I understand, constructors are just functions: https://go.dev/doc/effective_go#composite_literals

It seems that we are covered by this scope